.pre-commit-config.yaml

```
@@ -1,3 +1,9 @@
+# Integration with GitHub Actions
+# See https://pre-commit.ci/
+ci:
+    autofix_prs: false
+    autoupdate_schedule: monthly
```

I guess it prefers to do weekly `monthly` seems a bit leisurely :)
```suggestion
    autoupdate_schedule: weekly
```